Dockerfile

```diff
@@ -63,29 +62,40 @@ RUN HDF5_INCDIR=/usr/include/hdf5/serial
 RUN pip install --upgrade pip 
 RUN pip install -r requirements.txt
 
-# Install GDAL 2.4.1 from source
-RUN wget http://download.osgeo.org/gdal/2.4.1/gdal-2.4.1.tar.gz \
-    && tar -xzf gdal-2.4.1.tar.gz \
-    && rm gdal-2.4.1.tar.gz
+# TODO: not sure that we need this?
+# Install GDAL 3.5.2 from source
+# RUN wget http://download.osgeo.org/gdal/3.5.2/gdal-3.5.2.tar.gz \
+RUN wget https://ftp.osuosl.org/pub/osgeo/download/gdal/3.5.2/gdal-3.5.2.tar.gz \
+    && tar -xzf gdal-3.5.2.tar.gz \
+    && rm gdal-3.5.2.tar.gz
 
-WORKDIR /gdal-2.4.1
+WORKDIR /gdal-3.5.2
 RUN ./configure --with-python --with-geos=yes \
     && make \
     && sudo make install \
     && sudo ldconfig
 WORKDIR /
 
+# TODO: do we need to add Python bindings for Gdal?
+# https://mothergeo-py.readthedocs.io/en/latest/development/how-to/gdal-ubuntu-pkg.html
+# RUN python -m pip install --upgrade --no-cache-dir setuptools==58.0.2
+# RUN pip install GDAL==3.2.2
+# RUN pip install GDAL==`ogrinfo --version`
+
 # Install iRODS
 RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | sudo apt-key add - \
-    && echo "deb [arch=amd64] https://packages.irods.org/apt/ trusty main" | \
+    && echo "deb [arch=amd64] https://packages.irods.org/apt/ bullseye main" | \
     sudo tee /etc/apt/sources.list.d/renci-irods.list \
     && sudo apt-get update && sudo apt-get install -y \
     apt-transport-https \
     irods-runtime \
     irods-icommands
 
+# TODO: move this to requirements.txt not sure that we actually need it...
```

Comment:
I'm pretty confident both of these modules are necessary.  Depending on updates to the modules, we may no longer need to install from source.

Reply:
sounds good on keeping it, thanks
I tried installing from
https://pypi.org/project/google.foresite-toolkit/
just now and it doesn't work so keeping the build from your repo